source/scripts/models/GameContainer.js

```diff
@@ -65,13 +71,38 @@ export default class GameContainer extends Pixi.Container {
         this.music = new Audio(TRACK_1)
         this.music.loop = true
         this.music.volume = 0.5
-        this.music.play()
+        //this.music.play()
```

Comment:
Ha ha! Listening to the first 10 seconds of the song got a little annoying with every refresh, eh?

Eventually, we should wrap this in:

```js
if(STAGE == "PRODUCTION") {
    this.music.play()
}
```

Which will restrict the music to only final production builds, not our development builds.